backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/externalsystem/invokeExternalSystem.feature

```diff
@@ -41,3 +41,16 @@ Feature: external system invocation using metasfresh api
       | param.key | param.value |
       | OrderId   | 111         |
       | OrderNo   | 222         |
+
+
+  @from:cucumber
+  Scenario: The request is good and the external Other is invoked via the correct process
+    Given add Other external system config with identifier: otherConfigIdentifier
+      | Name | Value |
+      | Test | Test  |
+
+    And store endpointPath /api/externalsystem/Other/:otherConfigIdentifier/test in context, resolving placeholder as ExternalSystem.value
```

Comment:
is the `:` correct? shouldn't it be just `/api/externalsystem/Other/otherConfigIdentifier/test` ?

Reply:
i guess i got it wrongly..re-checking..